File: library/src/main/java/com/github/easyguide/client/ILayerChain.java
package com.github.easyguide.client;

import com.github.easyguide.layer.AbsGuideLayer;

/**
 * Created by shenxl on 2018/9/3.
 */
public interface ILayerChain {
    AbsGuideLayer getCurrentLayer();
    boolean hasNextLayer();
    void stepNext();
}

File: library/src/main/java/com/github/easyguide/client/ILayerChain.kt
package com.github.easyguide.client

import com.github.easyguide.layer.AbsGuideLayer

/**
 * Created by shenxl on 2018/9/3.
 */
interface ILayerChain {
    val currentLayer: AbsGuideLayer
    fun hasNextLayer(): Boolean
    fun stepNext()
}
